src/webview.ts

```diff
@@ -266,8 +267,8 @@ export class Webview {
         reqPtr: Deno.PointerValue,
         arg: Deno.PointerValue | null,
       ) => {
-        const seq = new Deno.UnsafePointerView(BigInt(seqPtr)).getCString();
-        const req = new Deno.UnsafePointerView(BigInt(reqPtr)).getCString();
+        const seq = seqPtr ? new Deno.UnsafePointerView(seqPtr).getCString() : '-1';
+        const req = reqPtr ? new Deno.UnsafePointerView(reqPtr).getCString() : '[]';
```

Comment:
Oh, good call out! I meant to add an explanation to the description.

Original empty pointer views would have resulted in an empty string. While rare if at all possible, I also knew it could cause an error when parsed as `args` in the `bind` callback.

Given more thought, this could have unintended side-effects if `bindRaw` is called for reasons other than `bind`. I'm now in favor of empty strings for the callback.